src/Symfony/Component/Messenger/Bridge/Redis/Transport/Connection.php

```diff
@@ -132,7 +133,7 @@ public function __construct(array $options, \Redis|Relay|\RedisCluster|null $red
                             if ($address = $sentinel->getMasterAddrByName($sentinelMaster)) {
                                 [$host, $port] = $address;
                             }
-                        } catch (\RedisException|\Relay\Exception $redisException) {
+                        } catch (\RedisException|RelayException $redisException) {
```

Comment:
looks like Psalm cannot handle this, should I bother by that and stop the PR or continue on it?
cc @xabbuh 

also, reported here: https://github.com/vimeo/psalm/issues/10840